tools/PI/DevHome.PI/Views/BarWindowVertical.xaml

```diff
@@ -55,6 +55,14 @@
                 Style="{StaticResource ChromeButton}" >
                 <TextBlock Text="{x:Bind _viewModel.CurrentSnapButtonText, Mode=OneWay}" FontSize="{StaticResource CaptionTextBlockFontSize}"/>
             </Button>
+            <Button
+                x:Uid="SwitchToLargeLayoutButton" 
```

Comment:
Maybe rename to ExpandCollapseButton (or something like that) to be more acuate to the behavior?